### PR TITLE
Xrddev 1821 ssMainpage streamlining

### PR DESCRIPTION
**Pull request by teemukin65**

### CODE REVIEW

File: src/proxy-ui-api/frontend/tests/e2e/pages/ssMainPage.js [hunk 1]
```diff
@@ -693,374 +135,75 @@ module.exports = {
...
-        saveButton: '//button[@data-test="save-service-parameters"]',
-        closeButton: '//button[@data-test="close"]',
-        urlHelp: '//div[@data-test="service-parameters-service-url-label"]//i',
-        timeoutHelp: '//div[@data-test="service-parameters-timeout-label"]//i',
-        verifyCertHelp: '//div[@data-test="service-parameters-verify-tls-label"]//i',
-        activeTooltip: '//div[contains(@class, "v-tooltip__content") and contains(@class,"menuable__content__active")]//span',
-        addSubjectsButton: '//button[@data-test="show-add-subjects"]',
-        removeAllButton: '//button[@data-test="remove-subjects"]',
-      },
+      commands: [operationDetailsCommands],
```
Comment (turkkaholmqvist) [inline src/proxy-ui-api/frontend/tests/e2e/pages/ssMainPage.js:180]: These arrays are not necessary as long as there is only one import inside of it - it's not an issue but i wonder whether we should establish some sort of common practice that array is always created regardless if it has only one member or other way around, never use array for single reference.

**Comment by iluwatar** [comment]
It's currently conflicting with `develop` branch

File: src/proxy-ui-api/frontend/vue.config.js [hunk 2]
```diff
@@ -3,7 +3,7 @@ const path = require('path');
 
 module.exports = {
   devServer: {
-    proxy: process.env.PROXY_ADDRESS || 'https://localhost:4100',
+    proxy: process.env.PROXY_ADDRESS || 'https://localhost:5005',
```
Comment (iluwatar) [inline src/proxy-ui-api/frontend/vue.config.js:6]: Is this change intentional?

**Review by iluwatar — APPROVED** [review]
Added one question there, but otherwise I think it's good to be merged once the conflicts have been addressed.

File: src/proxy-ui-api/frontend/vue.config.js (hunk 2 above)
Comment (teemukin65) [reply src/proxy-ui-api/frontend/vue.config.js:6]: No.. Needs to be fixed. Leaked due to local port mapping setup.

File: src/proxy-ui-api/frontend/tests/e2e/pages/ssMainPage.js (hunk 1 above)
Comment (teemukin65) [reply src/proxy-ui-api/frontend/tests/e2e/pages/ssMainPage.js:180]: I have seen these single-item array mandatory even thought not required by the documentation. If left out the test cases fail due to not finding those commands.

File: src/proxy-ui-api/frontend/vue.config.js (hunk 2 above)
Comment (teemukin65) [reply src/proxy-ui-api/frontend/vue.config.js]: fixed in : 1159ab7
